crates/std_detect/src/detect/os/x86.rs

```diff
@@ -119,7 +119,6 @@ pub(crate) fn detect_features() -> cache::Initializer {
         enable(extended_features_ebx, 19, Feature::adx);
         enable(extended_features_ebx, 11, Feature::rtm);
         enable(proc_info_edx, 4, Feature::tsc);
-        enable(proc_info_edx, 23, Feature::mmx);
```

Comment:
We want to keep feature detection for MMX.

Reply:
Test worked without that but sure I'll bring it back:
```
$ cargo test --package std_detect --test cpu-detection -- x86_all --exact --nocapture <

   Compiling libc v0.2.76
   Compiling cfg-if v0.1.10
   Compiling byteorder v1.3.4
   Compiling cupid v0.6.1
   Compiling auxv v0.3.3
   Compiling std_detect v0.1.5 (D:\tmp\stdarch\crates\std_detect)
    Finished test [unoptimized + debuginfo] target(s) in 12.19s  
     Running target\debug\deps\cpu_detection-3a542c0e111d2458.exe

running 1 test
aes: true     
pcmulqdq: true
rdrand: true  
rdseed: true  
tsc: true     
mmx: true
sse: true
sse2: true
sse3: true
ssse3: true
sse4.1: true
sse4.2: true
sse4a: true
sha: true
avx: true
avx2: true
avx512f: false
avx512cd: false
avx512er: false
avx512pf: false
avx512bw: false
avx512dq: false
avx512vl: false
avx512ifma: false
avx512vbmi: false
avx512vpopcntdq: false
avx512vbmi2 false
avx512gfni false
avx512vaes false
avx512vpclmulqdq false
avx512vnni false
avx512bitalg false
avx512bf16 false
avx512vp2intersect false
f16c: true
fma: true
bmi1: true
bmi2: true
abm: true
lzcnt: true
tbm: false
popcnt: true
fxsr: true
xsave: true
xsaveopt: true
xsaves: true
xsavec: true
test x86_all ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 1 filtered out
```

Reply:
That is because the test only tests that getting the feature compiles and doesn't panic. Because `cfg!(target_feature = "mmx")` returns true, it still shows up as true in the test output, but if the `cfg!(target_feature)` fast-path of `is_x86_feature_detected!` was removed, the test would give false for mmx.